vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/grid/GridWrap.java

```diff
@@ -521,4 +573,14 @@ private String getColumnInternalId(Grid.Column targetColumn) {
         }
     }
 
+    private String getColumnKey(Grid.Column targetColumn) {
```

Comment:
Can't find where this method is used. Is it here as a helper for other features?